packages/webview_flutter/webview_flutter_android/lib/src/android_webview_controller.dart

```diff
@@ -392,6 +395,19 @@ class AndroidWebViewWidgetCreationParams
   /// outside of tests.
   @visibleForTesting
   final InstanceManager instanceManager;
+
+  /// Whether the [WebView] will be displayed using the Hybrid Composition
+  /// PlatformView implementation.
+  ///
+  /// For most use cases, this flag should be set to false. Hybrid Composition
+  /// can have performance costs but doesn't have the limitation of rendering to
+  /// an Android SurfaceTexture. See
+  /// https://flutter.dev/docs/development/platform-integration/platform-views#performance
+  /// https://github.com/flutter/flutter/issues/104889,
+  /// https://github.com/flutter/flutter/issues/104889.
```

Comment:
These two are the same link.

Reply:
Whoops. That one is supposed to be https://github.com/flutter/flutter/issues/116954. Updated.